### PR TITLE
Layout Updates

### PR DESCRIPTION
**Pull request by ladomyriak**
Screenshot from 2019-06-28 13-03-27
Screenshot from 2019-06-28 13-03-58
Add full screen table width, small changes on the TopBar, loader aligned to center

### CODE REVIEW

File: src/pages/OverviewPage.js [hunk 1]
```diff
@@ -187,12 +187,12 @@ class OverviewPage extends Component {
...
         color="#f07"
-        style={{marginTop: '10em'}}
+        style={{margin: 'auto'}}
       />
     ) : (
       <main className={classes.main}>
-        <div className={classes.container}>
-          <div className={classes.filter}>
+        <div className={classes.container} style={{display: 'flex', alignItems: 'center', justifyContent: 'space-between'}}>
+          <div className={classes.filter} style={{zIndex: 100}}>
```
Comment (szist) [inline src/pages/OverviewPage.js]: We are already using css-in-js and you have the classes there. These shouldn't be directly passed as a style object, but instead added to the styling on line starting from 36.

File: src/components/OverviewTable.js [hunk 2]
```diff
@@ -93,6 +96,19 @@ class OverviewTable extends React.Component {
     return {cursor, scrollTo}
   }
 
+  countColumnsWidth() {
+    const vw = window.innerWidth - 75 // minus container inner offsets
```
Comment (szist) [inline src/components/OverviewTable.js]: what about scrollbars?

Comment (ladomyriak) [reply src/components/OverviewTable.js]: it includes 15px for scrollbar. If there is no scrollbar, the table has 15px offset on the right
Screenshot from 2019-07-09 12-27-29

File: src/pages/OverviewPage.js (hunk 1 above)
Comment (ladomyriak) [reply src/pages/OverviewPage.js]: It is already merged, should I create new PR to fix it, or it could be fixed with new changes later?

Comment (szist) [reply src/pages/OverviewPage.js]: feel free to create new PRs :) (I was on holiday last week)

File: src/components/OverviewTable.js (hunk 2 above)
Comment (szist) [reply src/components/OverviewTable.js]: hm... scrollbars are not really uniform across platforms - iOS sometimes uses overlayed scrollbars, firefox uses narrower, edge uses extra-wide ones. There are ways to detect the scrollbar width.

**Pull request merged**